docsrc/imap/developer/developer-testing.rst

```diff
@@ -51,6 +52,7 @@ Install and configure Cassandane
         sudo cpan -i XML::Fast
         sudo cpan -i Data::ICal::TimeZone
         sudo cpan -i Text::VCardFast
+        sudo cpan -i Email:MIME
```

Comment:
Ditto earlier comment -- this list of cpan commands is for stuff that wasn't available via the apt command above.